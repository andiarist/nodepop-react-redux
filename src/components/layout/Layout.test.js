import React from 'react';
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import Layout from './Layout';

describe('Testing <Layout />', () => {
  test('should show <Layout /> properly', () => {
    const props = {
      title: 'title',
      children: 'children',
    };
    const wrapper = shallow(<Layout {...props} />);

    expect(wrapper).toMatchSnapshot();
  });
});
