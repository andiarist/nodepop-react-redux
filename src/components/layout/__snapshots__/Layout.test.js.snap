// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <Layout /> should show <Layout /> properly 1`] = `
<Layout
  className="layout"
>
  <Header />
  <Content
    className="main"
  >
    <Row
      className="row"
    >
      <Col
        className="col"
        offset={4}
        span={16}
      >
        <Title
          level={2}
        >
          title
        </Title>
        <div>
          children
        </div>
      </Col>
    </Row>
  </Content>
  <Footer
    className="footer"
  >
    Â© 2020 Keepcoding
  </Footer>
</Layout>
`;
